model/task_lifecycle.go

```diff
@@ -293,21 +293,17 @@ func TryResetTask(ctx context.Context, settings *evergreen.Settings, taskId, use
 
 	maxExecution := evergreen.MaxTaskExecution
 
-	if evergreen.IsCommitQueueRequester(t.Requester) && evergreen.IsSystemFailedTaskStatus(t.Status) {
-		maxSystemFailedTaskRetries := settings.CommitQueue.MaxSystemFailedTaskRetries
-		if maxSystemFailedTaskRetries > 0 {
-			maxExecution = maxSystemFailedTaskRetries
-		}
-	}
-	// if we've reached the max number of executions for this task, mark it as finished and failed
-	if t.Execution >= maxExecution {
-		// restarting from the UI bypasses the restart cap
-		msg := fmt.Sprintf("task '%s' reached max execution %d: ", t.Id, maxExecution)
-		if origin == evergreen.UIPackage || origin == evergreen.RESTV2Package || origin == evergreen.GithubCheckRun {
```

Comment:
I probably did this but I see no reason for GithubCheckRuns to receive an exception.

Reply:
Oh no wait I didn't, this is behavior for the new project. I assume this should also be checked for the other checks below as well @bynn ? Should I just add this to UserTriggeredOrigins?

Reply:
Yes, since this is manually triggered by users clicking a button on the github UI!